modules/i18n/src/translations/en-US/portals/impersonation.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { ImpersonationNS } from "../../../models";
+
+export const impersonation: ImpersonationNS = {
+    description: "Configure impersonation settings for organization.",
+    form: {
+        enableEmailNotification: {
+            hint: "If enabled, an email notification will send to the impersonated user after impersonation occurred.",
```

Comment:
Shouldn't the hint text be changed as follows? afair, this setting sends a mail before the impersonation starts. Please correct me if I'm wrong.

```suggestion
            hint: "If enabled, an email notification will be sent to the impersonated user prior the impersonation starts.",
```

Reply:
this is a notification of impersonation started, hence it will be sent after impersonation occured.

Reply:
Ack, Still let's fix the grammar issues.

```suggestion
            hint: "If enabled, an email notification will be sent to the impersonated user after impersonation starts.
```

better if we can avoid passive voice.

```suggestion
            hint: "If enabled, the impersonated user will receive an email notification after impersonation starts.
```

cc: @himeshsiriwardana